THIRD-PARTY-NOTICES.TXT

```diff
@@ -521,3 +521,63 @@ COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
 IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 =========================================
+
+
+License notice for .NET Community Toolkit
+=========================================
```

Comment:
@PureWeen is it OK to take in code from CommunityToolkit/dotnet (EquatableArray) and dotnet/BenchmarkDotNet (HashCode)?